db/db.py

```diff
@@ -0,0 +1,28 @@
+from dotenv import load_dotenv
+import psycopg2
+import os
+
+load_dotenv()
+
+PGUSER = os.getenv("PGUSER")
+PGHOST = os.getenv("PGHOST")
+PGPORT = os.getenv("PGPORT")
+PGPASSWORD = os.getenv("PGPASSWORD")
+PGDATABASE = os.getenv("PGDATABASE")
+
+con = psycopg2.connect(
+  user=PGUSER,
+  host=PGHOST,
+  port=PGPORT,
+  password=PGPASSWORD,
+  database=PGDATABASE
+)
+
+cur = con.cursor()
+
+cur.execute("SELECT * FROM representatives")
+con.commit()
+
+rows = cur.fetchall()
+
+print(rows)
```

Comment:
This file feels more like an adhoc script rather than a module that can be used by any other piece of code. Considering adding this to a `scripts` directory instead of a `db` directory.